assets/src/edit-story/components/form/color/colorPreview.js

```diff
@@ -291,6 +291,7 @@ function ColorPreview({
           <VisualPreviewButton
             {...buttonProps}
             color={previewStyle?.backgroundColor}
+            tabIndex="-1"
```

Comment:
Hm, why are we removing this (and some other components) from the tab order? I designed this to be accessible in the first place (to the best of my efforts), and this seems to counter some of that?